digestbot/core/common/Enums.py

```diff
@@ -1,10 +1,18 @@
 from enum import Enum, unique
 
+# TODO: fix it
+# TODO: maybe https://stackoverflow.com/questions/37183612/how-to-define-a-mapping-of-enum-members-in-the-enum-type?
```

Comment:
Also we can do like this: 
```python
SortingType['replies'.upper()]
```

Reply:
Now we have a problem that column name is replies_count, but ask users to enter "replies_count" is worse than "replies". So now I added this dirty hack to merge subsystems, but it's better to solve it later: either rename the column in our database or create some intellectual mapper (and I think mapper should be near request parsing, not near the enum)